doc/tutorial/Client.lhs

```diff
@@ -88,7 +91,7 @@ the function `client`. It takes one argument:
 api :: Proxy API
 api = Proxy
 
-position :<|> hello :<|> marketing = client api
+position :<|> hello :<|> marketing :<|> emptyClient = client api
```

Comment:
I'd rather write `EmptyAPIClient` (or `EmptyClient` after rename) directly here and elaborate that you can do so (and it's safe) because it's a always succeeding irrefutable pattern match like:

```haskell
let (x, ()) = (42, ())
```